### PR TITLE
Make sewt.is-not.cool redirect to my self-hosted Apache server (proxied)

### PR DESCRIPTION
**Pull request by SkyExploreWasTaken**
Server IP (for lazy people that don't want to read the file) : 82.67.68.228

### CODE REVIEW

File: domains/sewt.json [hunk 1]
```diff
@@ -5,10 +5,8 @@
         "email": "contact@sewt.is-a.dev",
         "discord": "skyexplorewastaken"
     },
-
     "record": {
-        "CNAME": "sewt-website.onrender.com"
+        "A": ["82.67.68.228"],
```
Comment (CuteDog5695) [inline domains/sewt.json:9]:
```suggestion
        "A": ["82.67.68.228"]
```

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]
Mf thinks we don't check the file 💀

**Review by CuteDog5695 — APPROVED** [review]
Seems good to me now

**Comment by CuteDog5695** [comment]
I aint waiting for checks to finish

**Comment by SkyExploreWasTaken** [comment]
thx

**Pull request merged**